objectivec/README.md

```diff
@@ -146,6 +146,29 @@ supported keys are:
     of being plain `#import "some/path/file.pbobjc.h"` lines, they will be
     framework based, i.e. - `#import <VALUE/file.pbobjc.h>`.
 
+    _NOTE:_ If this is used with `named_framework_to_proto_path_mappings_path`,
+    then this is effectively the _default_ to use for everything that wasn't
+    mapped by the other.
+
+  * `named_framework_to_proto_path_mappings_path`: The `value` used for this key
+    is a path to a file containing the listing of framework names and proto
+    files. The generator use this to decide if another proto file referenced
+    should use a framework style import vs. a user level import
+    (`#import <FRAMEWORK/file.pbobjc.h>` vs `#import "dir/file.pbobjc.h"`).
+
+    The format of the file is:
+      * An entry is line of `frameworkName=file.proto, dir/file2.proto`.
```

Comment:
is this supposed to be `:` instead of `:`? Code below shows:

`int offset = line.find(':');`